packages/editor/src/editor/sidebar/fields/InputFieldWithBrowser.tsx

```diff
@@ -21,14 +21,13 @@ export const InputFieldWithBrowser = ({
 }: InputFieldProps & { browsers: Array<BrowserType>; options?: TextBrowserFieldOptions }) => {
   const [open, setOpen] = useState(false);
   const inputRef = useRef<HTMLInputElement>(null);
-  const cmsQuickFixPopoverRef = useRef<HTMLDivElement>(null);
   const { isFocusWithin, focusWithinProps } = useOnFocus(value, onChange);
   const { handleTextSelection, showQuickFix, getSelectedText, selection } = useTextSelection(inputRef);
 
   return (
     <Dialog open={open} onOpenChange={setOpen}>
       <BasicField label={label} message={message} style={{ flex: '1' }} {...focusWithinProps} className='badge-field' tabIndex={0}>
-        {isFocusWithin || cmsQuickFixPopoverRef.current ? (
```

Comment:
fyi @ivy-edp, @ivy-fhe 

It is not good to access ref values in the render part (beaks a rule of react)
So I wrote this stuff a little bit different. Should still be working as before

Reply:
:+1: goot to know

Reply:
I guess I need to take a closer look at these rules again 😬 😅 
https://react.dev/reference/rules

Reply:
wasn't aware of that case too, but this eslint plugin should check stuff like that :)